x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/config_form.tsx

```diff
@@ -25,14 +25,27 @@ const PolicyDetailCard = styled.div`
   }
 `;
 export const ConfigForm: React.FC<{
+  /**
+   * A subtitle for this component.
+   **/
   type: string;
-  supportedOss: string[];
+  /**
+   * Types of supported operating systems.
+   */
+  supportedOss: React.ReactNode;
```

Comment:
this was an array of translated strings. it was being concat'd with `, ` but that should be done via `i18n`. taking a react node instead.